library/vyos_l3_interfaces.py

```diff
@@ -122,249 +123,745 @@
 # Before state:
 # -------------
 #
-# vyos:~$ show configuration commands | grep -e eth[2,3]
-# set interfaces ethernet eth2 hw-id '08:00:27:c2:98:23'
-# set interfaces ethernet eth3 hw-id '08:00:27:43:70:8c'
-# set interfaces ethernet eth3 vif 101
-# set interfaces ethernet eth3 vif 102
-
-- name: Merge provided configuration with device configuration
+# vyos@vyos:~$ show configuration commands | grep interfaces
+# set interfaces ethernet eth0 address 'dhcp'
+# set interfaces ethernet eth0 hw-id '08:00:27:e9:f6:02'
+# set interfaces ethernet eth2 hw-id '08:00:27:c6:c1:3f'
+# set interfaces ethernet eth3 hw-id '08:00:27:38:bc:3f'
+# set interfaces loopback lo
+#
+- name:Merge provided configuration with device configuration
   vyos_l3_interfaces:
     config:
-      - name: eth2
+      - name:eth2
         ipv4:
-          - address: 192.0.2.10/28
-          - address: 198.51.100.40/27
+          - address:192.0.2.10/28
```

Comment:
Same comment as above for examples too. This will probably not be a valid task, iirc.